frontend/tests/Factories/Run.php

```diff
@@ -119,7 +128,9 @@ public static function createCourseAssignmentRun(
         $r = self::createRequestCourseAssignmentCommon(
             $problemData,
             $courseAssignmentData,
-            $participant
+            $participant,
+            null,
```

Comment:
Le puedes agregar:
```suggestion
            /*$login=*/ null,
```
? 
Esto nos sirve para tener un poco de contexto cuando mandamos algo no-descriptivo